src/main/resources/db/migration/V10__imrt_ddl.sql

```diff
@@ -0,0 +1,80 @@
+/*************************************************************
+ * Denormalize the data for better search performance
+ *************************************************************/
+
+ALTER TABLE item
```

Comment:
I think there needs to be a discussion around column sizing.  We'll need to narrow these down later this sprint.